cpp/include/raft/distance/fused_l2_nn.cuh

```diff
@@ -172,8 +172,7 @@ void fusedL2NN(OutT* min,
  *
  * @tparam DataT     data type
  * @tparam OutT      output type to either store 1-NN indices and their minimum
- *                   distances or store only the min distances. Accordingly, one
- *                   has to pass an appropriate `ReduceOpT`
+ *                   distances or store only the min distances.
```

Comment:
Nitpick: it would help to give an example of the output pair type:
```suggestion
 *                   distances (e.g. cub::KeyValuePair<int, float>) or store only the min distances.
```